core/execution/market.go

```diff
@@ -668,19 +668,19 @@ func (m *Market) PostRestore(ctx context.Context) error {
 
 	m.liquidity.ReconcileWithOrderBook(m.matching)
 
-	if err := m.peggedOrders.ReconcileWithOrderBook(m.matching); err != nil {
```

Comment:
Why don't we need to do this anymore?

Reply:
We do not store orders all the time, only when they are parked, which means they are not on the book then

Reply:
Ok I believe you, but I'm also nervous because this bug was a bastard to hunt down a fix at the time :p